src/overlay_params.cpp

```diff
@@ -674,14 +698,18 @@ static void set_param_defaults(struct overlay_params *params){
 
 void
 parse_overlay_config(struct overlay_params *params,
-                  const char *env)
+                  const char *env, bool use_existing_preset)
 {
-   *params = {};
+   std::vector<int> default_preset = {-1, 0, 1, 2, 3, 4};
```

Comment:
this doesn't account for user defined presets. There's no hard limit to how many can be defined

Reply:
Correct.  These just correspond with the current default presets found in `presets()`

Reply:
The user can specify how every many presets he wants